.github/workflows/e2e_test_ci.yml

```diff
@@ -575,103 +667,105 @@ jobs:
-        run: |
-          curl https://clickhouse.com/ | sh
-          sudo ./clickhouse install
-          sudo clickhouse server --config=/etc/clickhouse-server/config.xml --daemon  -- --tcp_port=9001
-          sleep 5
-          clickhouse client --port 9001 -udefault --query="CREATE USER test_user IDENTIFIED BY 'password';"
-          clickhouse client --port 9001 -udefault --query="GRANT ALL PRIVILEGES ON testdb.* TO 'test_user';"
-          clickhouse client --port 9001 -utest_user --password password --query="CREATE DATABASE IF NOT EXISTS testdb;"
-
-      - name: Wait for Clickhouse to start
-        run: sleep 10
-
-      - name: Check Clickhouse
-        run: clickhouse client --port 9001 -utest_user --password password --query="SELECT version();"
-
-      - name: Prepare Clickhouse dataset
-        run: |
-          clickhouse client --port 9001 -m -n -utest_user --password password --database=testdb < test/scripts/setup-data-clickhouse.sql
-          clickhouse client --port 9001 -utest_user --password password --database=testdb --query="SELECT * FROM test_clickhouse_table;"
-
-      - name: download artifacts - build_${{ matrix.target.target_os }}_${{ matrix.target.target_arch }}
-        uses: actions/download-artifact@v4
-        with:
-          name: build_${{ matrix.target.target_os }}_${{ matrix.target.target_arch }}
-          path: ./build
-
-      - name: Install spice
-        uses: ./.github/actions/install-spice
-        with:
-          build-path: ./build
-
-      - name: Init spice app
-        run: |
-          spice init test_app
-
-      - name: Spice dataset configure
-        working-directory: test_app
-        run: |
-          echo -e "test_clickhouse_table\neth recent blocks\nclickhouse:test_clickhouse_table\ny" | spice dataset configure
-          # configure clickhouse credentials
-          echo -e "params:\n  clickhouse_host: localhost\n  clickhouse_tcp_port: 9001\n  clickhouse_db: testdb\n  clickhouse_user: test_user\n  clickhouse_secure: false\n" >> ./datasets/test_clickhouse_table/dataset.yaml
-          cat spicepod.yaml
-          cat ./datasets/test_clickhouse_table/dataset.yaml
-
-      - name: Start spice runtime
-        env:
-          SPICE_CLICKHOUSE_PASS: password
-        working-directory: test_app
-        run: |
-          spice run &> spice.log &
-          # time to initialize added dataset
-          sleep 10
-
-      - name: Wait for Spice runtime healthy
-        working-directory: test_app
-        timeout-minutes: 1
-        run: |
-          while [[ "$(curl -s http://localhost:8090/health)" != "ok" ]]; do sleep 1; done
-
-      - name: Verify dataset
-        uses: ./.github/actions/verify-dataset
-        with:
-          name: test_clickhouse_table
-
-      - name: Stop spice and check logs
-        working-directory: test_app
-        if: always()
-        run: |
-          killall spice || true
-          cat spice.log
+  # TODO: Renable clickhouse when bugs are fixed
```

Comment:
Let's wait to merge this PR once my clickhouse fix is merged and we can re-enable this.